yaus_c4gt_frontend/src/components/layout/Sidenav.js

```diff
@@ -0,0 +1,282 @@
+import { Menu, Button } from "antd";
+import { NavLink, useLocation } from "react-router-dom";
+import logo from "../../assets/images/logo.png";
+
+function Sidenav({ color }) {
+  const { pathname } = useLocation();
+  const page = pathname.replace("/", "");
+
+  const dashboard = [
```

Comment:
Remove the svgs from here

Reply:
resolved in 80396e97b161ddd30d0022b950c212eb1adce878